scripts/blockchains/tezos.ts

```diff
@@ -188,6 +188,16 @@ export const tezosIosList: App[] = [
     logo: 'ios-trust.png',
     universalLink: 'https://trustwallet.com',
     deepLink: 'trust://'
+  },
+  {
+    key: 'exodus_mobile',
+    name: 'Exodus Mobile',
+    shortName: 'Exodus',
+    color: '',
+    logo: 'exodus.svg',
+    supportedInteractionStandards: ['beacon'],
+    deepLink: 'exodus://wc?uri=',
```

Comment:
If `?type=tzip10&data=23RuZM...` is appended then it should probably be changed to:
```suggestion
    deepLink: 'exodus://wc',
```
I'm assuming that the wallet will be opened with `exodus://wc?type=tzip10&data=23RuZM...`